resources/buildPlatformMap.properties

```diff
@@ -10,6 +10,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# NOTE: These mappings are used exclusively for the "Re-run in Grinder"
+#       links in the console log output
+
```

Comment:
You should delete this comment, as it's not only for rerun link (also used for exclusion etc). This file should match whatever is defined in https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/openjdk_tests

Reply:
OK ... Shelley's earlier comment had said:

> buildPlatformMap.properties is used for rerun in Grinder link alone

but I can remove it.

Reply:
my misunderstanding... Now I need to know how are we passing other tests without this change and will have to look and understand it better, dangit! ;)

Reply:
I guess no one is using arm_linux for exclude, so there's no break till now :)

Reply:
I think it is not necessary to add this comment, it is also used by other things such as internal TKG and etc.